pyaerocom/aeroval/coldatatojson_engine.py

```diff
@@ -889,14 +991,44 @@ def _process_heatmap_data(data, region_ids, use_weights, use_country,
                                                                    per,
                                                                    season)
 
+                            if add_trends:
+                                # Calculates the start and stop years. min_yrs have a test value of 7 years. Should be set in cfg
+                                start_stop = per.split("-")
+                                if len(start_stop) == 2:
+                                    (start, stop) = [int(i) for i in start_stop]
+                                else:
+                                    start = stop = int(start_stop[0])
+
+                                if stop - start >= trends_min_yrs:
+                                    subset_time_series = subset.get_regional_timeseries(regid)
+                                    (obs_trend,
+                                     mod_trend) = _make_trends_from_timeseries(
+                                        subset_time_series["obs"],
+                                        subset_time_series["mod"],
+                                        freq,
+                                        season,
+                                        start,
+                                        stop,
+                                        trends_min_yrs
+                                        )
```

Comment:
Are there any scenarios where this could crash, perhaps put it in a try / except block so that not the whole function crashes? (please check also for other possible occasions above / below).

Reply:
Should I just pass if that fails, or give some error/warning?